src/main/java/seedu/address/logic/commands/TagDeleteCommand.java

```diff
@@ -142,71 +139,9 @@ private String generateDeleteMessage(Person personToEdit, Person editedPerson, M
             }
         }
 
-        deletePersonInWedding(editedPerson, model, tagsToDelete);
+        model.deletePersonInWedding(editedPerson, model, tagsToDelete);
         return String.format(MESSAGE_DELETE_TAG_SUCCESS, Messages.tagSetToString(tagsToDelete),
                 Messages.format(editedPerson), Messages.format(editedPerson),
                 Messages.tagSetToString(tagsToDelete));
     }
-
```

Comment:
Good job moving these methods to Model & ModelManager! Can I check if this has been done for TagAddCommand as well? I think it has setPersonInWedding, updatePersonInWedding and getWeddingFromTags, which should all be transferred to Model & ModelManager as well